_posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md

```diff
@@ -376,6 +376,151 @@ ENDPOINT=http://localhost:8080 CONFIG_ENDPOINT=http://localhost:19071 poetry run
 < DELETE http://localhost:19071/application/v2/tenant/default/application/default - 200
 {% endhighlight %}
 
+### Weaviate
+Weaviate is a vector search engine specifically designed for natural language and numerical data. It uses contextualized embeddings in data objects to understand semantic similarity. Currently, it supports only Hierarchical Navigable Small World(HNSW) indexing and is more costly on building data to indexes however has a fast query time and high scalability. Weaviate is open-source, easy to use, flexible, extensible and has a Contributor License Agreement.
```

Comment:
Add a space before `(HNSW)` and an oxford comma before ", and is more costly" and ", and has a CLA". Make "However, it claims fast query time ..." a sentence.